charts/orchestrator/templates/sonataflows.yaml

```diff
@@ -32,6 +32,9 @@ spec:
         limits:
           memory: {{ .Values.orchestrator.sonataPlatform.resources.limits.memory }}
           cpu: {{ .Values.orchestrator.sonataPlatform.resources.limits.cpu }}
+  {{- if .Values.monitoring.enabled }}
+  monitoring: true
```

Comment:
this isn't part of the platform's spec:
https://github.com/apache/incubator-kie-kogito-serverless-operator/blob/main/api/v1alpha08/sonataflowplatform_types.go#L34

Reply:
@JudeNiroshan is there a requirement to contribute a feature for the sonataflow operator for this?

Reply:
Yes, there is an epic in Sonataflow Operator -> https://github.com/apache/incubator-kie-kogito-serverless-operator/issues/461

But it is not ready yet. Maybe we can ignore this CR condition until we have the operator modification.

Reply:
I think we should wait for the epic to be completed and available for testing before introducing the monitoring attribute to this PR.
We can keep this one as draft for the meantime. WDYT?

Reply:
Yeah, I agree with you. I have already marked this as a draft PR. Once I have the serverless-workflows changes merged, I can do the manual testing on an already deployed helm chart.(Without the CR if condition)